docs/zkapps/advanced-snarkyjs/merkle-tree.mdx

```diff
@@ -13,31 +13,30 @@ Please note that zkApp programmability is not yet available on Mina Mainnet, but
 
 #### Referencing off-chain data
 
-We already learned that zkApps can only store a small amount of data on-chain, the reason for that is we want Mina to stay succinct and not become bloated.
+zkApps can store only a small amount of data on-chain so that Mina stays succinct and does not become bloated.
```

Comment:
Suggestion `zkApp accounts can store only a limited amount of data on chain so that Mina's chain remains succinct and does not become bloated`

(The difference is that `zkApps` are defined as `the smart contract + the UI`. The `zkApp account` is the on-chain aspect that can store data on chain.)